packages/web3-eth-ens/src/lib/ResolverMethodHandler.js

```diff
@@ -70,7 +70,7 @@ ResolverMethodHandler.prototype.call = function (callback) {
     var promiEvent = new PromiEvent();
     var preparedArguments = this.parent.prepareArguments(this.ensName, this.methodArguments);
 
-    this.parent.registry.resolver(this.ensName).then(function (resolver) {
+    this.parent.registry.getResolver(this.ensName).then(function (resolver) {
```

Comment:
Required change or we have to use the deprecated and badly named ``resolver`` method